test/mock_data.py

```diff
@@ -155,6 +162,11 @@
     "thumbnail_base64": "UklGRjQAAABXRUJQVlA4ICgAAADQAQCdASoCAAEAAUAmJYwCdAEO/gOOAAD+qlQWHDxhNJOjVlqIb8AA",
 }
 
+IMAGE_GET_METADATA_ALL_VALUES_AFTER_PATCH = {
+    **IMAGE_GET_METADATA_ALL_VALUES,
+    **IMAGE_PATCH_METADATA_DATA_ALL_VALUES,
+}
```

Comment:
```suggestion
IMAGE_GET_METADATA_DATA_ALL_VALUES_AFTER_PATCH = {
    **IMAGE_GET_METADATA_DATA_ALL_VALUES,
    **IMAGE_PATCH_METADATA_DATA_ALL_VALUES,
}
```
Perhaps a bit redundant but its the data that forms the metadata. Could you do the same in the other places, we seem to have 8 with _DATA, and 6 without.